core/src/components/refresher-content/refresher-content.tsx

```diff
@@ -51,7 +52,7 @@ export class RefresherContent implements ComponentInterface {
   componentWillLoad() {
     if (this.pullingIcon === undefined) {
       const mode = getIonMode(this);
-      const overflowRefresher = (this.el.style as any).webkitOverflowScrolling !== undefined ? 'lines' : 'arrow-down';
+      const overflowRefresher = (this.el.style as any).webkitOverflowScrolling !== undefined ? 'lines' : arrowDown;
```

Comment:
Wouldn't `lines` need to be swapped out with the SVG content as well?

Reply:
"lines" is a spinner not an ion-icon, and the spinner SVG data is defined inside of Ionic (not ionicons): https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/spinner/spinner-configs.ts#L83